src/SDL2.cs

```diff
@@ -2091,6 +2201,14 @@ SDL_BlendMode blendMode
 			byte b,
 			byte a
 		);
+		
+		public static int SDL_SetRenderDrawColor(
```

Comment:
It's in render.h, and documented [here](https://wiki.libsdl.org/SDL_SetRenderDrawColor)